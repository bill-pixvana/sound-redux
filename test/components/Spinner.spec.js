import React from 'React';
import Spinner from '../../client/components/Spinner';
import { shallow } from 'enzyme';

describe('Spinner', () => {
  it('should render correctly', () => {
    const result = shallow(<Spinner />);
    expect(result).toMatchSnapshot();
  });
});
