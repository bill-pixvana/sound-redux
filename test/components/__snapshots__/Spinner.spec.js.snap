// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render correctly 1`] = `
<div
  className="spinner-container"
>
  <div
    className="spinner"
  >
    <div
      className="rect1"
    />
    <div
      className="rect2"
    />
    <div
      className="rect3"
    />
    <div
      className="rect4"
    />
    <div
      className="rect5"
    />
  </div>
</div>
`;
